deploy/cephfs/kubernetes/csi-provisioner-rbac.yaml

```diff
@@ -77,10 +77,6 @@ metadata:
   namespace: default
   name: cephfs-external-provisioner-cfg
 rules:
-  # remove this once we stop supporting v1.0.0
-  - apiGroups: [""]
-    resources: ["configmaps"]
-    verbs: ["get", "list", "create", "delete"]
```

Comment:
same changes are required in helm charts and for the rbd.

Reply:
I am keeping this PR for just cleanup commit and will put RBAC one different.. @Madhu-1